src/components/team/Teammate.jsx

```diff
@@ -1,5 +1,34 @@
+import React from "react";
+import Image from "next/image";
+
 const Teammate = () => {
-  return <div>Teammate</div>;
+  return (
+    <div className="bg-gradient-to-r from-hlg-blue-300 to-hlg-black ...">
+      <div className="flex">
+        <div className="mr-50">
+          <Image
+            src="team/graves.png" // Path to your image inside the 'public' directory
+            alt="Graves Image"
+            width={500} // Desired width of the image
+            height={300} // Desired height of the image
+          />
+        </div>
+
+        <div className="mt-10 ml-10">
+          <div className="bg-gradient-to-r from-hlg-blue-100 to-hlg-blue-200 bg-clip-text text-transparent">
+            <div className="font-bold leading-tight text-5xl -mb-3">
+              <p>Matthew &quot;GetTurtled&quot;</p>
```

Comment:
make the name more bold like it is in the image https://tailwindcss.com/docs/font-weight